src/app/repository-management/repository-overview/repository-overview.component.ts

```diff
@@ -63,8 +63,9 @@ export class RepositoryOverviewComponent implements OnInit {
     }
 
     navigateToRepo(): void {
+        const url = new URL(this.repoURL)
         window.open(
-            this.repoURL,
+            url.protocol + '//' + url.host + '/',
```

Comment:
This code reads like #WTF. Why can't the plain repoURL not be used? - Is the variable name `repoURL` wrong? Should it be `repoApiUrl`? (The casing is using [Google's style guide](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case))

Reply:
This should now be fixed here: https://github.com/OpenTOSCA/ui/commit/dc6e3482d131c2f88c0ff383b44bb6646ab79f9d
I completely reworked the repoURL variable to a repositoryApiUrl variable which now is type URL.